src/Property.php

```diff
@@ -19,7 +19,7 @@ class Property extends ReflectionProperty
     protected $hasTypeDeclaration = false;
 
     /** @var bool */
-    protected $isNullable = false;
+    protected $isNullable = true;
```

Comment:
Why did you change the default value to `true`? I prefer this package to be non-nullable by default, als it prevents lots of bugs.

Reply:
As explained in PR: to be consistent with test `empty_type_declaration_allows_everything`.
If it is left `false`, then this test breaks.

Reply:
Ok, will take a look